app/controllers/integrated/add_tax_member_controller.rb

```diff
@@ -6,6 +6,18 @@ def additional_model_attributes
       }
     end
 
+    def update_models
+      member_data = combined_attributes(member_params)
+      if member_data[:relationship] == "spouse"
+        member_data[:married] = "yes"
+        current_application.navigator.update!(anyone_married: true)
+        current_application.primary_member.update!(married: "yes")
+        member_data[:tax_relationship] = member_data[:tax_relationship_spouse]
```

Comment:
This is icky, but I couldn't think of an alternative way to keep the two checkbox sets distinct for accessibility purposes while still using the Follow Up pattern that GCF established.

Reply:
yeah, that's tricky! I think it's easy to read, though, and the tests add some clarity.